README.md

```diff
@@ -146,7 +146,7 @@ yargs(Deno.args)
 As of `v16`,`yargs` supports ESM imports:
 
 ```js
-import yargs from 'yargs'
+import * as yargs from 'yargs'
```

Comment:
I believe this is only necessary if you're using TypeScript?

And the syntax can vary depending on how you have the compiler configured, e.g.,

https://github.com/googleapis/repo-automation-bots/blob/master/packages/owl-bot/src/bin/commands/copy-code.ts#L18

I think we document the appropriate approach here:

https://github.com/yargs/yargs/blob/master/docs/typescript.md

Could we draw attention to this better in the README?